### PR TITLE
Add

### PR DESCRIPTION
**Pull request by albin1616**
add function implemented

### CODE REVIEW

File: add.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+function add(a,b){
```
Comment (Rajharsh2) [inline add.js]: - Provide Jsoc documentation for better understanding.

**Review by Rajharsh2 — CHANGES_REQUESTED** [review]

File: add.js (hunk 1 above)
Comment (albin1616) [reply add.js]: resolved

File: add.js [hunk 2]
```diff
@@ -1,3 +1,9 @@
+/**
+ * 
+ * @param {number} a must be a number
+ * @param {number} b munst be a number
+ * @returns the result
+ */
 function add(a,b){
```
Comment (manvithalva12) [inline add.js]: please change the function name.

**Review by manvithalva12 — CHANGES_REQUESTED** [review]

File: add.js (hunk 1 above)
Comment (Rajharsh2) [reply add.js]: look good to me

**Review by Rajharsh2 — APPROVED** [review]

File: add.js (hunk 2 above)
Comment (albin1616) [reply add.js]: resolved

Comment (albin1616) [reply add.js]: resloved

Comment (albin1616) [reply add.js]: i resloved again

**Review by manvithalva12 — APPROVED** [review]
done

**Pull request merged**